modules/ROOT/partials/configuration/max_height.adoc

```diff
@@ -17,6 +18,4 @@ tinymce.init({
 });
 ----
 
-____
-*Note*: If you set the option link:editor-size-options.html#resize[`+resize+`] to `+false+` the resize handle will be disabled and a user will not be able to resize the editor (by manual dragging). Note that `+resize+` defaults to `+false+` when the `+autoresize+` plugin is enabled.
-____
+NOTE: If you set the option xref:editor-size-options.adoc#resize[`+resize+`] to `+false+` the resize handle will be disabled and a user will not be able to resize the editor (by manual dragging). Note that `+resize+` defaults to `+false+` when the `+autoresize+` plugin is enabled.
```

Comment:
good to know it converts .adoc to .html for you during compile

Reply:
Yes, this is how internal page linking is meant to be done. If it's done this way it can also tell you when you have invalid links. This is one of the things I've mentioned we need to fix. See https://docs.antora.org/antora/latest/page/xref/